ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2402,7 +2402,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
 
                     break;
                 case StrategyTradeWorkFlowState.GoLongValidationRejected:
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode || !IsOnStrategyTradeWorkFlowStateEntryRejectionError) return ProcessWorkFlow(StrategyTradeWorkFlowState.Waiting);

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode  use the same paths a lot of the time-